packages/db/drizzle/0018_codex_schema.sql

```
@@ -0,0 +1,151 @@
+      START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1
+    );
+  END IF;
+  PERFORM setval('codex.user_submissions_id_seq',
+    GREATEST((SELECT last_value FROM codex.user_submissions_id_seq),
+             COALESCE((SELECT MAX(id) FROM codex.user_submissions), 0)) + 1, false);
+END $$;
+
+-- ─── Indexes ──────────────────────────────────────────────────────────────────
+
+CREATE INDEX IF NOT EXISTS idx_entry_references_target_entry_id
+  ON codex.entry_references USING btree (target_entry_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)